docs/assets/themes/zeppelin/js/toc.js

```diff
@@ -0,0 +1,98 @@
+// https://github.com/ghiculescu/jekyll-table-of-contents
```

Comment:
@Leemoonsoo Yeah I added the license info as you said. Please see [f704b5e](https://github.com/apache/zeppelin/pull/1031/commits/f704b5e6eacd04ac7e94e41736e42dd6b8af46ba).